ezstaking/chains.json

```diff
@@ -316,6 +316,15 @@
         "run_time": "every 5 minutes",
         "minimum_reward": 1000000
       }
+    },
+    {
+      "name": "tori",
```

Comment:
```suggestion
      "name": "teritori",
```

Reply:
Thank you!
Don't hesitate to accept the changes directly next time :p

Reply:
Haha I would but Github wouldn't let me - I think there might be a setting in your account that prevents it

Reply:
Oh right, I'll have to check this out, thanks again :)